src/javascript/app/common/guide.js

```diff
@@ -43,16 +44,12 @@ const Guide = (() => {
             $(opt.guideBtnID).remove();
             return;
         }
-
         makeButton();
     };
 
-    /*
-     *  do not show the guide button if its close (X) has been clicked before
-     */
     const isDisabled = () => {
         const disabled = Cookies.get(cookie_name);
-        return !!disabled && $.inArray(opt.script, disabled.split(',')) >= 0;
+        return !disabled && $.inArray(opt.script, disabled.split(',')) >= 0;
```

Comment:
we still want to hide it for the current user. just after logging out and logging in we want it to show again. also now for logged out clients it shows a grey box on the trade page.

Reply:
I implemented this kind of behaviour and updated code.